backend/src/api/routes/tag-route.ts

```diff
@@ -19,7 +19,7 @@ router
   .post(
     '/',
     permit(RoleType.ADMIN),
-    run((req) => create({ workspaceId: req.workspaceId, name: req.body.name })),
+    run((req) => create(req.workspaceId, { name: req.body.name })),
```

Comment:
Will not it be more neat and easier to read?
```suggestion
    run((req) => create(req.workspaceId, req.body.name )),
```